src/lib/components/drip-list-card/components/voting-round-splits.svelte

```diff
@@ -6,25 +6,26 @@
   import { fade } from 'svelte/transition';
 
   export let votingRound: VotingRound & { splits?: SplitsComponentSplitsReceiver[] };
+  export let maxRows: number | undefined = undefined;
 </script>
 
 <TransitionedHeight transitionHeightChanges>
   <div class="results" style:min-height={!votingRound.result ? '16rem' : undefined} out:fade>
     {#if !votingRound.result && votingRound.status === 'started'}
       <div class="empty-state" in:fade>
-        <Emoji emoji="🫙" size="huge" />
-        <h4>No recipients yet</h4>
-        <p>Collaborators are currently voting on the recipients of this Drip List.</p>
+        <Emoji emoji="🗳️" size="huge" />
+        <h4>Awaiting nominations</h4>
+        <p>No one has voted yet.</p>
```

Comment:
wouldn't call this "awaiting nominations" given we're actually going to add a separate nominations feature soon.

Reply:
k, changed to "Awaiting votes" in [e5e4ced](https://github.com/drips-network/app/pull/1042/commits/e5e4cedb05c4c2aea88b9942ad8ae8666d21d9ab)